modules/virt-example-vf-host-services.adoc

```diff
@@ -9,7 +9,7 @@ Update host network settings for Single Root I/O Virtualization (SR-IOV) network
 
 You can apply a `NodeNetworkConfigurationPolicy` manifest to an existing cluster to complete the following tasks:
 
-* Configure QoS or MTU host network settings for VFs to optimize performance.
+* Configure QoS host network settings for VFs to optimize performance.
```

Comment:
@cgoncalves - I removed the mention of MTU from the first bullet. However, can the next two bullets be retained or should they be removed? Thanks.

Reply:
The second one (add/remove/update VFs) has to be removed.

On the third one, could we be more explicit? We successfully tested change of bonding `mode` and `miimon` parameters.